### PR TITLE
Enrichment for extract indicators playbook v2

### PR DESCRIPTION
**Pull request by ssokolovich**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/20376

## Description
Added additional step for previewing pdf files as an image inside the war-room. For those images - the "PDF_images" tag was added as part of this step.

## Screenshots
Extract Indicators from file V2

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2.yml [hunk 1]
```diff
@@ -301,14 +312,6 @@ tasks:
                       value:
                         simple: text/rtf
                     ignorecase: true
-                - - operator: isNotEqualString
```
Comment (David-BMS) [inline Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2.yml:304]: @ssokolovich Please review why these changes to the filters occurred

**Comment by ShirleyDenkberg** [comment]
@David-BMS Doc review completed.

**Review by David-BMS — APPROVED** [review]
